docs-ref-services/iot-hub.md

```diff
@@ -24,17 +24,80 @@ Azure IoT Hub is a fully managed service that enables reliable and secure bidire
 - Provides extensive monitoring for device connectivity and device identity management events.
 - Includes device libraries for the most popular languages and platforms.
 
-## Install the modules with npm
-
 Use npm to install the Azure IoT Hub modules for Node.js
 
-### Management
+## Management Package
+
+### Install the npm packages

```

Comment:
Our modifications to other docs used the text `Install npm module`. Should we switch to use `packages` here instead?